### PR TITLE
rootfs-version-check: Added mender_boot_part_hex

### PR DESCRIPTION
**Pull request by ckelloug2**
Setting mender_boot_part_hex was missing from the places where the script set mender_boot_part.  Used printf to create a hex version of mender boot part and set mender_boot_part_hex to it.

### CODE REVIEW

**Comment by lluiscampos** [comment]
@drewmoseley you are listed as the maintainer of this module, could you please have a look?

**Comment by lluiscampos** [comment]
@ckelloug2 Hello and thanks for your contribution! While we wait for @drewmoseley review on the code, I can point you out that the commit needs to be signed according to [our contributing guideline](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md#sign-your-work).

**Review by drewmoseley — APPROVED** [review]
LGTM

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
Thanks @drewmoseley, I can take care of the formalities.

@ckelloug2 please sign your commit and we are good to go here :+1:

**Pull request merged**
